### DISCUSSION TITLE
I made a codemod to convert from js-yaml to yaml; any interest?

### DISCUSSION THREAD

**Discussion by danbeam** [eemeli/yaml#568] [Show and tell]
Hey there,

I made a codemod (via jscodeshift) to change callsites across a codebase from using `js-yaml` to `yaml` and was wondering if you'd be interested in my upstreaming it?

It doesn't handle every little edge case (and there are some things that aren't really easily done in a codemod dealing with minor incompatibilities between the two libraries), but it helped me convert a lot of files quickly and easily.

Let me know!

**Comment by eemeli** [comment]
Sounds like a thing that could/should be hosted separately. If it looks good, we could add a link to it from the docs.

**Reply by danbeam** [reply]
OK, I'll create a thing you can evaluate / possibly link to soon then (hoping next week).

**Comment by danbeam** [comment]
By the way @eemeli, I wrote this (small) comemod while working for a company and I'm going through their whole process for seeing whether I'm allowed to open source it, etc.

If there's too many snags, I may rewrite it on my own time / computer / own resources, etc.

That's the hold up here.

**Reply by danbeam** [reply]
So uh, I know it's been forever, but I'm leaving that company and the thing probably won't get open sourced.

So I'll rewrite it on my own time / resources when I'm out (soon, like in the next few weeks).

I think I can also make the codemod two way (`js-yaml` ⇔ `yaml`) so even more of a reason to host it somewhere intermediate I suppose.
